kore/src/Kore/Internal/Conditional.hs

```diff
@@ -146,54 +145,6 @@ instance TopBottom term
     isBottom Conditional {term, predicate, substitution} =
         isBottom term || isBottom predicate || isBottom substitution
 
-instance
-    ( SortedVariable variable
-    , Ord variable
-    , Show variable
-    , Unparse variable
-    , Unparse child
-    ) =>
-    Unparse (Conditional variable child)
-  where
-    unparse Conditional { term, predicate, substitution } =
-        unparseAnd
-            (below "/* term: */" (unparse term))
-            (unparseAnd
-                (below
-                    "/* predicate: */"
-                    (unparse predicate)
```

Comment:
...we should use `Kore.Syntax.Predicate.fromPredicate` on the `predicate` to set the sort (to the value of `termLikeSort term`)...